Source/Scene/Cesium3DTileFeature.js

```diff
@@ -92,17 +116,301 @@ define([
          */
         color : {
             get : function() {
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    return label.fillColor;
+                }
+
                 if (!this._color) {
                     this._color = new Color();
                 }
                 return this._batchTable.getColor(this._batchId, this._color);
             },
             set : function(value) {
-                this._batchTable.setColor(this._batchId, value);
+                if (defined(this._labelCollection)) {
+                    var label = this._labelCollection.get(this._batchId);
+                    label.fillColor = value;
+                    setBillboardImage(this);
+                } else {
+                    this._batchTable.setColor(this._batchId, value);
+                }
+            }
+        },
+
+        /**
+         * Gets and sets the point size of this feature.
+         * <p>
+         * Only applied when the feature is a point feature and <code>image</code> is <code>undefined</code>.
+         * </p>
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Number}
+         */
+        pointSize : {
+            get : function() {
+                return this._pointSize;
+            },
+            set :function(value) {
+                this._pointSize = value;
+                if (defined(this._billboardCollection)) {
```

Comment:
Is it worth optimizing for special cases when a `PointPrimitiveCollection` could be used?  Seems like it would be hard to manage when setting some properties at runtime could cause it to split into a billboard collection.  Add a `PERFORMANCE_IDEA` / roadmap item if you think this would be reasonable and useful for some cases.